crdschurch/crds-net
# update resi-player for SBOP

## Task

[Asana Task](https://app.asana.com/0/1201454420722044/1203822148391333/f)

## Solution
Update `resi-player.js` times to:

SATURDAY:
Live Stream Player appears Saturday at 2:55pm and come down 7pm Saturday.
/watch
location pages
Green banner added during same times 

SUNDAY:
NO Live stream player anywhere
NO green "message live" alert 
Sarah will publish an orange alert in contentful linking directly to the message recording once it is uploaded by Crds people at 8am on Sunday!

### Notes
The week before SBOP we will want to add this to the superbowl page in the CMS:
```
{% include _resi-player.html %}
```

## Tester Notes
To test:
1. Go to your settings and set the date to the live streaming times
2. Then go to the location pages and /superbowl2023 (superbowl page currently does not the resi-player)
3. You should then see the a live stream player at the top of the page

## Corresponding PRs
https://github.com/crdschurch/crds-unified/pull/486
https://github.com/crdschurch/crds-components/pull/1850

## Preview
[Deploy Preview](https://deploy-preview-2841--crds-net.netlify.app/oakley)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203749593820297
